.github/file-filters.yml

```diff
@@ -73,6 +73,19 @@ frontend:
   - packages/grid/frontend/**/*.ts
   - packages/grid/frontend/**/*.tsx
 
+syftjs:
+  - .github/workflows/pr-tests-syftjs.yml
+  - "**.json"
+  - "**.js"
+  - "**.ts"
+  - "**.yaml"
+  - "**.lock"
```

Comment:
We shouldn't listen to all these filetypes. It's enough to hit packages/syftjs for now.